cl/search/models.py

```diff
@@ -1186,6 +1186,17 @@ def pacer_url(self):
                     )
                 )
 
+    @property
+    def pacerdash_url(self) -> str:
+        """Construct the PacerDash URL, if we can. Else, return None."""
+
+        if self.pacer_doc_id:
+            return (
+                "https://www.pacerdash.com/court-listener-checkouts/"
```

Comment:
Two small quibbles. 

One I think this could use better docstrings.

two.  I think you could return None and simplify the method.  

I may be wrong but I would do this 

```
@property
    def pacerdash_url(self) -> Optional[str]:
        """Construct the PacerDash URL, if we can. Else, return None."""

        if not self.pacer_doc_id: return None
        return (
            "https://www.pacerdash.com/court-listener-checkouts/"
            f"{self.pacer_doc_id}"
        )
```

Reply:
Improved docstring and try to change the return None into one line `if not self.pacer_doc_id: return None` as suggested but seems that `black` didn't like it because it reformatted into two lines.
```
if not self.pacer_doc_id:
     return None
```

but it works!